pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarRecordCursor.java

```diff
@@ -560,6 +563,18 @@ public boolean advanceNextPosition() {
         return true;
     }
 
+    private SchemaInfo getBytesSchemaInfo(SchemaType schemaType, String schemaName) {
+        if (!schemaType.equals(SchemaType.BYTES) && !schemaType.equals(SchemaType.NONE)) {
+            return null;
+        }
+        if (schemaName.equals(Schema.BYTES.getSchemaInfo().getName())) {
```

Comment:
Because of the schema types of the `Schema.BYTES ` and `Schema.BYTEBUFFER` all are `SchemaType.BYTES`, so I use the schema name to make the comparison.

Reply:
Okay. Make sense.